packages/clay-css/src/content/form_custom.html

```diff
@@ -263,6 +292,16 @@ <h3>Custom File</h3>
 		$(input).val(name);
 	});
 
+	$('.custom-range-progress .custom-range').on('change', function(event) {
```

Comment:
Just a hint of change that makes the `range-progress` movement more fluid.
```suggestion
	$('.custom-range-progress .custom-range').on('input', function(event) {
```

Reply:
Feel free to adapt in any way. I think this is just sample code :)